docs/user-guide/settings/executors.md

```
@@ -4,110 +4,112 @@
 > **前端**：`frontend/src/components/settings/ExecutorsPanel.tsx`
 > **后端**：`backend/src/handlers/executor_config.rs`
 
-「执行器」（Executor）是 ntd 把 Todo 实际跑起来的桥梁。ntd 本身不执行代码，而是把 Todo 的 prompt 交给外部 CLI 工具（Claude Code、Codex、Hermes、Kimi 等）来跑。
+「执行器」（Executor）是 ntd 把 Todo实际跑起来的桥梁。ntd本身不执行代码，而是把 Todo 的 prompt交给外部 CLI工具（Claude Code、Codex、Hermes、Kimi 等）来跑。
 
 ---
 
-## 1. 支持的执行器
+##1. 支持的执行器
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)